### PR TITLE
Creates devise user and homepage

### PR DESCRIPTION
**Pull request by carolineartz**
@jmhinric please review

### CODE REVIEW

File: app/assets/javascripts/welcome.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (jmhinric) [inline app/assets/javascripts/welcome.coffee]: Unneeded file

File: app/helpers/welcome_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
```
Comment (jmhinric) [inline app/helpers/welcome_helper.rb:1]: Unneeded file

File: spec/controllers/welcome_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (jmhinric) [inline spec/controllers/welcome_controller_spec.rb]: Unneeded file

File: spec/controllers/welcome_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
+RSpec.describe WelcomeController, type: :controller do
```
Comment (jmhinric) [inline spec/controllers/welcome_controller_spec.rb]: Unneeded file

File: spec/helpers/welcome_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (jmhinric) [inline spec/helpers/welcome_helper_spec.rb]: Unneeded file! 😝

Comment (carolineartz) [reply spec/helpers/welcome_helper_spec.rb]: good call 😭

**Pull request merged**